### ISSUE TITLE
Click handlers do not receive the event objects which makes it difficult to identify the click type

### ISSUE THREAD

**Issue by Giridharan2112** [nicoespeon/gitgraph.js#129]
Hi,
    We are in need of having multiple behaviors for left and right click separately. To differentiate the click types, we would need the event object being passed to the click handlers. I have made the change locally and have tried it out. If you are open to accept changes in this regard, I shall go ahead and create a PR.

**Comment by nicoespeon** [comment]
Hi @Giridharan2112!

Passing the original event object to handlers sounds legit to me, you just presented a perfect use case for such need.

If you have something, please feel welcome to open a PR − otherwise I'll give it a look, that's an easy one, but I'm currently working on #67.

**Comment by Giridharan2112** [comment]
Hi @nicoespeon!

Thanks for accepting the feature request. I have the changes for it. I will raise a PR for it by EOD.

**Comment by Giridharan2112** [comment]
Can you please provide me access to push a branch?

**Comment by nicoespeon** [comment]
@Giridharan2112 I think you should fork the repo, push to yours and create a PR from here. That's usually how I do for that kind of contribution to a lib on GitHub, see https://help.github.com/articles/fork-a-repo/ =)

Thanks a lot 👍

**Comment by Giridharan2112** [comment]
Thanks for letting me know the means to do it. Have created a PR #130

**Comment by nicoespeon** [comment]
Hi @Giridharan2112,

It's all merged, thanks for your contribution.
I just published [v1.5.0](https://github.com/nicoespeon/gitgraph.js/releases/tag/v1.5.0) which includes these changes.

A little note however: to prevent backward incompatible changes, I changed the signature of your implementation so the `event` object is the last parameter of the callback. See 5777e1f01ac76d7bb7c74d38951184306bcc2dcb.
